src/behaviors/index.ts

```diff
@@ -33,14 +31,15 @@ const HYPERVIEW_BEHAVIORS = [
   HvUnselectAll,
 ];
 
-export const getRegistry = (behaviors: HvBehavior[] = []): BehaviorRegistry =>
-  [...HYPERVIEW_BEHAVIORS, ...behaviors].reduce(
+export const getRegistry = (behaviors: HvBehavior[] = []): BehaviorRegistry => {
```

Comment:
Necessary?

Reply:
I need to investigate what eslint rule causes this, it's happened in most files. I'll update before merging.

Reply:
I ran into it a bunch in my work. I think it was a conflict between eslint and prettier which didn't allow single line arrow statements.

Reply:
todo: revert

Reply:
a93387d